kernel-kit/build.sh

```diff
@@ -855,7 +856,7 @@ else
 	export MAKE_TARGETS="bzImage modules"
 fi
 echo "$MAKE ${JOBS} ${MAKE_TARGETS}
-$MAKE INSTALL_MOD_PATH=${linux_kernel_dir} modules_install" > compile ## debug
+$MAKE INSTALL_MOD_PATH=${linux_kernel_dir}/usr modules_install" > compile ## debug
```

Comment:
You know I'm not against change but Slackware will likely never adopt the /usr merge.

What about this in `build.conf`

```cfg
## for best compatibility with debian, ubuntu, arch; set the UM variable
UM=/usr
## slackware, salix etc
# UM=''
```

Then all the lines in `build.sh` where `/usr` is required can be thus:

```
$MAKE INSTALL_MOD_PATH=${linux_kernel_dir}${UM} modules_install" > compile ## debug
```